src/block/block_open.c

```diff
@@ -44,7 +44,7 @@ __wt_block_manager_create(WT_SESSION_IMPL *session, const char *filename, uint32
         if ((ret = __wt_open(session, filename, WT_FS_OPEN_FILE_TYPE_DATA,
                WT_FS_OPEN_CREATE | WT_FS_OPEN_DURABLE | WT_FS_OPEN_EXCLUSIVE, &fh)) == 0)
             break;
-        WT_ERR_TEST(ret != EEXIST, ret);
+        WT_ERR_TEST(ret != EEXIST, ret, true);
```

Comment:
All three of these new args appear to be backward and should send in `false` instead. (Or you could change the polarity of the argument and change the macros instead - either way works.)

Reply:
Yup. My bad.  I've fixed those.